app/models/game.rb

```diff
@@ -1,4 +1,53 @@
 class Game < ActiveRecord::Base
   belongs_to :users
   has_many :pieces
+  after_create initialize_board!
+
+  def initialize_board!
+    #white pieces
+    7.times do |n|
+        Pawn.create(
+            piece_id: id,
+            color: true,
+            y_cord: 1,
+            x_cord: n,
+            type: pawn
```

Comment:
Good point, I'm going to refactor this and double check with Ken pertaining to where we should specify it